frontend/components/utils/FormatCurrency.vue

```diff
@@ -1,19 +1,24 @@
 <template>
-    <span
-        >{{ sign }}{{ formattedNumber }}<span class="uppercase"> {{ currency }}</span></span
-    >
+    <span>
+        {{ sign }}
+        <animated-number v-if="isValidNumber" :value="value" :decimal-places="decimals" />
```

Comment:
Since `{{ sign }}` and `<animated-number` are on different lines, when rendered, they will have space in-between, while they had none before

Reply:
Fixed this!